### PR TITLE
fix: include index{.js,.d.ts} in published types package

### PR DESCRIPTION
**Pull request by mattwebbio**
## Description

Right now, only `dist/lib/src` and `dist/lib/types` are included in the packed package, meaning that actually no code is in the published package. This small changes includes all files output to the `dist/lib` directory.

## Testing

_Steps taken to verify the changes_

## Checklist

I have:
* [x] Added new automated tests for any new functionality
* [x] Run `npm run build`

## Licensing statement (do not modify)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by msrose** [comment]
Hey @mattwebbio can you please clarify what "no code in published packages" means? Looking at the current dist for 0.3.0 it seems like all the correct files exist:

```
∫ msrose (1) ~/.nvm/versions/node/v18.14.2/lib/node_modules/@aws/universal-test-runner git:(4054bd7) ∫
Σ ls
LICENSE              NOTICE               README.md            THIRD_PARTY_LICENSES dist                 node_modules         package.json
∫ msrose (1) ~/.nvm/versions/node/v18.14.2/lib/node_modules/@aws/universal-test-runner git:(4054bd7) ∫
Σ ls -R dist
lib   types

dist/lib:
bin src

dist/lib/bin:
ErrorCodes.js cli.js

dist/lib/src:
ProtocolEnvVars.js ProtocolLogger.js  index.js           loadAdapter.js     log.js             mapEnvToResult.js  readProtocol.js    run.js

dist/types:
src

dist/types/src:
ProtocolEnvVars.d.ts ProtocolLogger.d.ts  index.d.ts           loadAdapter.d.ts     log.d.ts             mapEnvToResult.d.ts  readProtocol.d.ts    run.d.ts
```

Are you seeing errors on your end with the Node.js API being unable to import, or similar? Right now I think your change would only add the tests and test types to dist, which we should avoid doing.

**Comment by mattwebbio** [comment]
Hmm, that doesn't seem quite right. Here's what I'm seeing when installed via NPM in a project:
```
$ ~/src/cli/node_modules/@aws/universal-test-runner-types > npm pkg get version
"0.3.0"
$ ~/src/cli/node_modules/@aws/universal-test-runner-types > ls -R dist
lib

dist/lib:
index.js
```

And when building locally:
```
$ ~/src/universal-test-runner/packages/universal-test-runner-types > npm run clean && npm run compile
...
$ ~/src/universal-test-runner/packages/universal-test-runner-types > ls -R dist
lib   types

dist/lib:
index.js

dist/types:
index.d.ts
```

If I'm understanding correctly, tests wouldn't be included after the change this PR introduces because your `tsconfig.json` doesn't include a tests directory (let alone the fact there are no tests for the `types` package):
https://github.com/aws/universal-test-runner/blob/0ce391d4f001b0f2826bacf5278fdb31d6123942/packages/universal-test-runner-types/tsconfig.json#L18

It's worth noting that even if the hierarchy you're seeing locally is correct, you'd still need to change the `main` and `types` values in your `package.json`:
https://github.com/aws/universal-test-runner/blob/0ce391d4f001b0f2826bacf5278fdb31d6123942/packages/universal-test-runner-types/package.json#L5-L6

**Comment by mattwebbio** [comment]
Ah, I see what's happening. You're in `universal-test-runner`, and I'm in `universal-test-runner-types`.

**Comment by msrose** [comment]
Ahh sorry totally missed that, thanks 😅 I think this hasn't cropped up before since the package has only been a dev dependency within this monorepo. LGTM

File: packages/universal-test-runner-types/package.json [hunk 1]
```diff
@@ -11,8 +11,7 @@
     "prepack": "npm run clean && npm run compile"
   },
   "files": [
-    "dist/lib/src",
-    "dist/types/src",
+    "dist/lib",
```
Comment (msrose) [inline packages/universal-test-runner-types/package.json:14]: Should we just include all of `dist`? The file structure I'm seeing locally is:
```
dist/
  lib/
    index.js
  types/
    index.d.ts
```
so I think if we only include dist/lib, we won't get the index.d.ts file in the tarball.

**Comment by msrose** [comment]
Just left an additional question after double checking the local file structure :)

File: packages/universal-test-runner-types/package.json (hunk 1 above)
Comment (mattwebbio) [reply packages/universal-test-runner-types/package.json:14]: 🤦 yes

Comment (mattwebbio) [reply packages/universal-test-runner-types/package.json]: FIxed. Glad you're actually reading the PR 😆

Comment (msrose) [reply packages/universal-test-runner-types/package.json]: #229 FYI

**Comment by msrose** [comment]
0.3.1 published

**Pull request merged**